jest_ui.config.js

```diff
@@ -0,0 +1,52 @@
+/** @flow
+ * @format */
+
+const defaultPlatform = 'android';
+const rnPlatform = process.env.TEST_RN_PLATFORM || defaultPlatform;
+if ( process.env.TEST_RN_PLATFORM ) {
+	// eslint-disable-next-line no-console
+	console.log( 'Setting RN platform to: ' + process.env.TEST_RN_PLATFORM );
+} else {
+	// eslint-disable-next-line no-console
+	console.log( 'Setting RN platform to: default (' + defaultPlatform + ')' );
+}
+
+module.exports = {
```

Comment:
It seems that this Jest config file is mostly identical to the `jest.config.js` file so, can we layer them to avoid the code duplication? A way to do that is shown in the `jest_gb.config.js` file where we import the base one and only override the fields that need to be overriden.

Reply:
Yeah you're right! I've gone ahead and made that change